### ISSUE TITLE
meta_train_84

### ISSUE THREAD

**Issue by qinzhengmei** [ArmanAfrasiyabi/associative-alignment-fs#2]
Thank you for your excellent work!
I would like to know where to download the meta_train_84 dataset

**Comment by ArmanAfrasiyabi** [comment]
Thanks for your interest! As explained in Readme/Datasets, it can be downloaded from [here](https://drive.google.com/file/d/13ngR5yQLGlXjGqZSNYbXOKQF6iDTaVkp/view)

**Comment by qinzhengmei** [comment]
Thank you for your reply. 
I have some questions about the calculation method of  the centroid ui of Xrb, in the paper is written about the centroid of the set of related base examples (eq. 2), why in the code is calculated is the centroid of the  novel class examples.

def centroid_aligner(args, net, data, support_size, query_size, device):
    [xt, xa] = data    
    za = net(xa)        
    anchor = net(xt)   

    yt = torch.arange(0, args.test_n_way).view(args.test_n_way, 1, 1).expand(args.test_n_way, query_size, 1).long()
    yt = Variable(yt, requires_grad=False).to(device)
    
    anchor = anchor.view(args.test_n_way, support_size, -1)             
    za = za.view(za.size(0), -1)
 
    anchor_mu = anchor.mean(1)       

    dists = -euclidean_dist(za, anchor_mu)   
    log_exp_ndist = torch.nn.functional.log_softmax(dists, dim=1)     
    log_exp_ndist = log_exp_ndist.view(args.test_n_way, query_size, -1)    
    
    gather_log_exp_dist = -log_exp_ndist.gather(2, yt)                       
    loss = gather_log_exp_dist.squeeze().view(-1).mean()  
    
    return loss

Can you explain that
